packages/trader/src/App/Containers/trade-settings-extensions.tsx

```diff
@@ -19,15 +24,17 @@ const ChartSettingContainer = Loadable({
 //     loading: UILoader,
 // });
 
-const renderItemValue = (props, store) => (
-    <MobxContentProvider store={store}>
+const renderItemValue = <T extends object>(props: T, store: TCoreStores) => (
+    <TraderProviders store={store}>
         <ChartSettingContainer {...props} />
-    </MobxContentProvider>
+    </TraderProviders>
 );
 
-const TradeSettingsExtensions = ({ populateSettingsExtensions, store }) => {
+const TradeSettingsExtensions = observer(({ store }: TTradeSettingsExtensionsProps) => {
```

Comment:
@jim-deriv Same here.

Reply:
https://github.com/binary-com/deriv-app/pull/8270/files#r1177827285🙂

Reply:
As explained above, I'll base on the conclusion from you guys (@maryia-deriv and @farzin-deriv) to implement this change.